### ISSUE TITLE
Include Stash untracked files in the Commit Details View

### ISSUE THREAD

**Issue by thmatot** [mhutchie/vscode-git-graph#208]
It would be nice to add untrack files to stash files list (add with git stash -u)

**Comment by mhutchie** [comment]
Thanks for raising this improvement request!

Given the below example of how stashes are stored:

```
*-. 4444444 working changes
|\ \
| | * 3333333 untracked files
| * 2222222 index
|/
* 111111 base
```

The Commit Details / Comparison View's runs a `git diff` command to get the diffs between the stash (4444444) & its base (1111111). It's because the untracked files are stored in a commit with no parents (3333333) that it's not included it the `git diff` output, and consequently in Commit Details / Comparison Views.

I'll have a look into how best to achieve including the untracked files in the Commit Details / Comparison Views.

**Comment by mhutchie** [comment]
I've worked out how to achieve including untracked files from Stashes in the Commit Details View, by using information from the reflog and running an additional `git diff` command (specifically to get the stashes untracked files).

I'll have this available in the next release.

**Comment by mhutchie** [comment]
This will be available in v1.19.0.

If you'd like to use it before the next release, you can download [v1.19.0-beta.1](https://github.com/mhutchie/vscode-git-graph/releases/tag/v1.19.0-beta.1), and install it following the instructions provided [here](https://github.com/mhutchie/vscode-git-graph/wiki/Downloading-&-Installing-Releases#manual-installation).
